examples/plot_spm_auditory.py

```diff
@@ -83,9 +97,10 @@
         nib.save(out_map, map_path)
         print("\t\t%s map: %s" % (dtype, map_path))
 
-    # plot one activation map
+    #########################################################################
+    # Plot thresholded z scores map
     display = plot_stat_map(z_map, bg_img=mean_img, threshold=3.0,
                             display_mode='z', cut_coords=3, black_bg=True,
                             title=contrast_id)
 
-show()
+    show()
```

Comment:
I put it inside the loop for visual reasons, maps looked cluttered and ugly when displayed together if I remember correctly. Anyway finally only one contrast is being displayed.